File: app/src/main/java/com/example/watering/investrecord/DBHelper/SpendCashDBHelper.java
package com.example.watering.investrecord.DBHelper;

import android.content.Context;

/**
 * Created by watering on 17. 10. 21.
 */

@SuppressWarnings("DefaultFileTemplate")
public class SpendCashDBHelper extends DBHelper {

    public SpendCashDBHelper(Context context) {
        super(context);

        TABLE_NAME = "tbl_spend_cash";
        COLUMNS = new String [] {"spend_code TEXT UNIQUE","id_account INTEGER"};
    }
}

File: app/src/main/java/com/example/watering/investrecord/DBHelper/SpendCashDBHelper.kt
package com.example.watering.investrecord.DBHelper

import android.content.Context

/**
 * Created by watering on 17. 10. 21.
 */

class SpendCashDBHelper(context: Context) : DBHelper(context) {

    init {

        TABLE_NAME = "tbl_spend_cash"
        COLUMNS = arrayOf("spend_code TEXT UNIQUE", "id_account INTEGER")
    }
}
